action.yml

```diff
@@ -34,6 +34,7 @@ runs:
 
     - name: Checkout target branch
       uses: actions/checkout@v3
+      if: ${{ !env.ACT }}
```

Comment:
What's ACT and when it will contain a value?

Reply:
The action is tested with act-js and Mock GitHub, this is an env var only available when running tests with Act. There are limitations with act-js/Mock GitHub, specifically it does not allow for mocking steps within a composite action (only workflows), hence the use of stubbed steps. There is a project idea for an improved GitHub tester that would allow for mocking action steps, and these `env.ACT` checks could then be removed.